src/main/java/lottogame/view/InputView.java

```diff
@@ -28,21 +35,22 @@ public int getBonusNumberInput() {
 
     private User doGetUserInput() {
         System.out.println("구입 금액을 입력해 주세요.");
-        User user = new User(new Money(scanner.nextInt()));
+        Money money = new Money(scanner.nextInt());
+        System.out.println("");
```

Comment:
```suggestion
        System.out.println();
```

empty string은 굳이 입력하지 않아도 되겠네요~